src/helpers/hotkeysManager.js

```diff
@@ -6,7 +6,7 @@ import openFilePicker from 'helpers/openFilePicker';
 import copyText from 'helpers/copyText';
 import setToolModeAndGroup from 'helpers/setToolModeAndGroup';
 import { zoomIn, zoomOut } from 'helpers/zoom';
-import { print } from 'helpers/print';
+import print from 'src/apis/print';
```

Comment:
Our convention is that the internal code shouldn't use any exposed APIs.
So I think it'd be better if we still use
```
import { print } from 'helpers/print';
```
and then pass all the needed state into this helper function